src/main/Elevator.java

```diff
@@ -216,9 +227,10 @@ public void run() {
 		            tempElevator.setDoorsOpen(false);
 		        }
 		    };
+		    
 		    Timer timer = new Timer("Timer");
-		    long delay = fe == null ? 500 : 
-		    	fe.getErrorCode() == 1 ? 1000 : 500;
+		    long delay = fe == null ? Configurations.TIME_TO_LOAD_UNLOAD: 
+		    	fe.getErrorCode() == 1 ? 1000 : Configurations.TIME_TO_LOAD_UNLOAD;
```

Comment:
@krishane2018 can you double check this is correct and should we make change the 1000 into ```Configurations.TIME_TO_LOAD_UNLOAD * 2```?

Reply:
fixed it.